src/types/formatted/healthServices.ts

```diff
@@ -0,0 +1,13 @@
+export interface HealthService {
+  name: string
+  vetCenterTypeOfCare: string | null
```

Comment:
do all these formatted fields need `| null` or can they be rewritten as `vetCenterTypeOfCare?: string`

Reply:
I understand it to be functionally the same either way. will change